bin.src/create_agn_db.py

```diff
@@ -277,26 +280,64 @@ def get_m_i(abs_mag_i, redshift):
-                          + '"p": {"seed": %d, "agn_tau": %.3e, "agn_sfu": %.3e, ' % (ss, tt, sfu)
-                          + '"agn_sfg": %.3e, "agn_sfr": %.3e, "agn_sfi": %.3e, ' % (sfg, sfr, sfi)
-                          + '"agn_sfz": %.3e, "agn_sfy": %.3e}}' % (sfz, sfy))
-                     for ii, hh, mm, ss, tt, sfu, sfg, sfr, sfi, sfz, sfy in
-                     zip(galaxy_id, htmid, mag_norm, seed_arr, tau,
-                         sf_dict['u'], sf_dict['g'], sf_dict['r'], sf_dict['i'],
-                         sf_dict['z'], sf_dict['y']))
-
-            cursor.executemany('INSERT INTO agn_params VALUES(?, ?, ?, ?)', vals)
-            connection.commit()
+            varParamStr_format = '{"m": "applyAgn", "p": {"seed": %d, "agn_sf_u": %.3e, "agn_sf_g": %.3e, "agn_sf_r": %.3e, "agn_sf_i": %.3e, "agn_sf_z": %.3e, "agn_sf_y": %.3e, "agn_tau_u" : %.3e, "agn_tau_g" : %.3e, "agn_tau_r" : %.3e, "agn_tau_i" : %.3e, "agn_tau_z" : %.3e, "agn_tau_y" : %.3e}}'
+
+            row_by_row = zip(galaxy_id, htmid, mag_norm, seed_arr,
+                             sf_dict['u'], 
+                             sf_dict['g'], 
+                             sf_dict['r'], 
+                             sf_dict['i'],
+                             sf_dict['z'], 
+                             sf_dict['y'],
+                             tau_dict['u'],
+                             tau_dict['g'],
+                             tau_dict['r'],
+                             tau_dict['i'],
+                             tau_dict['z'],
+                             tau_dict['y'])
+
+            vals = []
+            for gal_id_i, htmid_i, mag_norm_i, seed_arr_i, 
+                sf_u, 
+                sf_g, 
+                sf_r, 
+                sf_i, 
+                sf_z, 
+                sf_y, 
+                tau_u, 
+                tau_g, 
+                tau_r, 
+                tau_i, 
+                tau_z, 
+                tau_y in row_by_row:
+                vals.append((int(gal_id_i), int(htmid_i), mag_norm_i,
+                             varParamStr_format % (seed_arr_i,
+                                                   sf_u, 
+                                                   sf_g, 
+                                                   sf_r, 
+                                                   sf_i, 
+                                                   sf_z, 
+                                                   sf_y, 
+                                                   tau_u, 
+                                                   tau_g, 
+                                                   tau_r, 
+                                                   tau_i, 
+                                                   tau_z, 
+                                                   tau_y)
+                             ))
+
+                cursor.executemany('INSERT INTO agn_params VALUES(?, ?, ?, ?)', vals)
```

Comment:
This line and the one below is indented one block too far.  It should be outside of the immediately preceding `for` loop.

Reply:
Fixed!